apps/right-to-rent-check/index.js

```diff
@@ -5,13 +5,27 @@ const controllers = require('hof-controllers');
 module.exports = {
   name: 'right-to-rent-check',
   baseUrl: '/right-to-rent-check',
+  params: '/:action?/:id?',
   steps: {
     '/': {
       controller: controllers.start,
```

Comment:
Yeah... it's a bit difficult at the moment because *so much* has just moved in hof core, so trying to start a new project *right now* is going to be tricky.

Swapping out the old things for the new things should be fairly straightforward though.